java/postgresql-jdbc/Portfile

```diff
@@ -23,25 +23,26 @@ distname            ${name}-${version}.src
 worksrcdir          ${name}-${version_major}.${version_minor}-${version_patch}.src
 
 master_sites        ${homepage}/download/
-checksums           rmd160  215bcc04ce3aa41266fc1fcb27a31a95a710c7bc \
-                    sha256  97ee866689f0e6c45f3e71bcb88f42f16ea9a8a79965e6790efb15e133585029 \
-                    size    1404018
+checksums           rmd160  017982ae8dfd0beb6e78befd45ee1181fbc84c97 \
+                    sha256  65b2385b7b627bb7369ce9d22d1d92a1c928a74eff93a120504c236f7685883c \
+                    size    1567710
 
 java.version        1.8*
 java.fallback       openjdk8
-depends_build       port:maven32
+depends_build       port:gradle
 
 use_configure       no
 
-build.cmd           mvn32
-build.target        package
-build.args          --define skipTests \
-                    --define maven.repo.local=${workpath}/m2/repository
+build.cmd           ${prefix}/bin/gradle
+build.target        assemble
+build.args          "--gradle-user-home=${workpath}/gradle \
+                     --project-prop=pgjdbc.skip.jre6 \
+                     --project-prop=pgjdbc.skip.jre7"
```

Comment:
We don't normally use quotes here. Are you doing that for some reason?

Reply:
Nope. 

I have another version of this Portfile with an experimental test phase. As those args are common to the build and test phases, they were set up with a:

```tcl
set gradle_args    "--gradle-user-home=${workpath}/gradle \
                    --project-prop=pgjdbc.skip.jre6 \
                    --project-prop=pgjdbc.skip.jre7"
```

I removed the variable from the stripped down version but forgot to remove the quotes...